newIDE/app/src/MainFrame/EditorContainers/HomePage/LearnSection/MainPage.js

```diff
@@ -0,0 +1,232 @@
+  onCreateProject: () => void,
+  onTabChange: (tab: HomeTab) => void,
+  onOpenHelpFinder: () => void,
+  onSelectCategory: (?TutorialCategory) => void,
+  tutorials: Array<Tutorial>,
+|};
+
+const MainPage = ({
+  onOpenOnboardingDialog,
+  onCreateProject,
+  onTabChange,
+  onOpenHelpFinder,
+  onSelectCategory,
+  tutorials,
+}: Props) => {
+  const classes = useStyles();
+  const windowWidth = useResponsiveWindowWidth();
+  const showTourHelpItem = !electron && !isMobile() && !isUserflowRunning;
+  const helpItems = [
+    showTourHelpItem
+      ? {
+          title: <Trans>Guided Tour</Trans>,
+          description: (
+            <Trans>Learn the fundamentals of the editor in 5 minutes</Trans>
+          ),
+          action: () => {
+            sendOnboardingManuallyOpened();
+            onOpenOnboardingDialog();
+          },
+        }
+      : undefined,
+    {
+      title: <Trans>Documentation</Trans>,
+      description: <Trans>Find the complete documentation on everything</Trans>,
+      action: onOpenHelpFinder,
+    },
+    {
+      title: <Trans>Examples</Trans>,
+      description: <Trans>Have look at existing games from the inside</Trans>,
+      action: onCreateProject,
+    },
+    {
+      title: <Trans>Community</Trans>,
+      description: <Trans>Ask your questions to the community</Trans>,
+      action: () => onTabChange('community'),
+    },
+  ].filter(Boolean);
+
+  return (
+    <SectionContainer title={<Trans>Help and guides</Trans>}>
+      <SectionRow>
+        <Line noMargin>
+          <GridList
+            cols={getColumnsFromWidth(windowWidth, showTourHelpItem)}
+            style={styles.grid}
+            cellHeight="auto"
+            spacing={10}
+          >
+            {helpItems.map((helpItem, index) => (
+              <GridListTile
+                key={index}
+                style={styles.gridListTile}
+                classes={{ tile: classes.tile }}
+              >
+                <CardWidget onClick={helpItem.action} key={index} size="small">
+                  <div style={styles.helpItem}>
+                    <Column alignItems="center">
+                      <Text size="block-title">{helpItem.title}</Text>
+                      <Text size="body" color="secondary">
+                        {helpItem.description}
+                      </Text>
+                    </Column>
+                  </div>
+                </CardWidget>
+              </GridListTile>
+            ))}
+          </GridList>
+        </Line>
+      </SectionRow>
+      <>
+        <SectionRow>
+          <LineStackLayout
+            justifyContent="space-between"
+            alignItems="center"
+            noMargin
+            expand
+          >
+            <Column noMargin>
+              <Text size="title">
+                <Trans>Guides and tutorials</Trans>
+              </Text>
+            </Column>
+            <Column noMargin>
+              {windowWidth === 'large' && (
```

Comment:
It's hidden in medium & small.
Mostly because it messes up the design (huge button) 
+ I don't see people accessing the app on mobile wanting to submit their game as an example